### DISCUSSION TITLE
Defining bearing in UCS analysis

### DISCUSSION THREAD

**Discussion by SquawkCodeVFR** [petrobras/ross#970] [Q&A] [answered]
Hello,

When using the UCS analysis for variable bearing stiffness, is there any possibilty in defining which bearing should be used for the calculation?

I have a rotor with two bearings and I would like to keep the first bearing at a constant stiffness and the second one should be used
with variable stiffness for the UCS calculation.

Thanks for your help!

**Comment by raphaeltimbo** [comment, marked as answer]
Hi @SquawkCodeVFR !
To do that you would need to modify the code in the rotor_assembly.py file, changing how the bearings list is built:
https://github.com/petrobras/ross/blob/3895ca66b22f7e859956a732bba72a3846d1d03d/ross/rotor_assembly.py#L2188-L2196

**Reply by SquawkCodeVFR** [reply]
Thank you so much! It worked for me with the following changes:

![bearing](https://user-images.githubusercontent.com/132356626/235702288-5665929f-d310-4192-90ce-92c61a069f19.PNG)
